test/functional/apps/dashboard/group5/dashboard_panel_listing.ts

```diff
@@ -17,7 +17,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const kibanaServer = getService('kibanaServer');
   const dashboardAddPanel = getService('dashboardAddPanel');
 
-  describe('dashboard panel listing', () => {
+  describe('dashboard panel listing', function () {
+    this.tags('skipFIPS');
```

Comment:
>[!TIP]
> Here and for other skips - it would be helpful to have a comment with a justification, and to track this somewhere if we intend to unskip them eventually.

Reply:
I think a follow up PR to 1) add comments to all skipped tests 2) try to fix the FTR FIPS Role overrides would be worthwhile.

Reply:
Sounds good to me. As I mentioned in https://github.com/elastic/kibana/pull/192632#discussion_r1842614539, it would only be worth adding comments if we can clearly articulate the problem to help future readers. Otherwise, it’s fine to keep it as is.